### PR TITLE
Release 3.0.2

### PR DESCRIPTION
**Pull request by shaun-nx**
Documentation updates for Release 3.0.2

### CODE REVIEW

File: docs/content/technical-specifications.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ We explicitly test the NGINX Ingress Controller (NIC) on a range of Kubernetes p
 {{% table %}}
 | NIC Version | Supported Kubernetes Version | NIC Helm Chart Version | NIC Operator Version | NGINX / NGINX Plus version |
 | --- | --- | --- | --- | --- |
-| 3.0.1 | 1.26 - 1.21 | 0.16.1 | 1.3.0 | 1.23.3 / R28 |
+| 3.0.2 | 1.26 - 1.21 | 0.16.1 | 1.3.0 | 1.23.3 / R28 |
```
Comment (shaun-nx) [inline docs/content/technical-specifications.md:26]: Will re-add 3.0.1 versions

Comment (ciarams87) [reply docs/content/technical-specifications.md]: You don't need to! We just keep the latest of each major version in this table

File: docs/content/technical-specifications.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ We explicitly test the NGINX Ingress Controller (NIC) on a range of Kubernetes p
 {{% table %}}
 | NIC Version | Supported Kubernetes Version | NIC Helm Chart Version | NIC Operator Version | NGINX / NGINX Plus version |
 | --- | --- | --- | --- | --- |
+| 3.0.2 | 1.26 - 1.21 | 0.16.2 | 1.3.0 | 1.23.3 / R28 |
 | 3.0.1 | 1.26 - 1.21 | 0.16.1 | 1.3.0 | 1.23.3 / R28 |
```
Comment (ciarams87) [inline docs/content/technical-specifications.md:27]:
```suggestion
```
We only need the latest major version in this table - the script updating the previous 3.0.1 version is by design.

File: docs/content/technical-specifications.md (hunk 1 above)
Comment (haywoodsh) [reply docs/content/technical-specifications.md]: I think we will have to update the helm chart and operator version though.

File: docs/content/app-protect-dos/configuration.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ docs: "DOCS-580"
 ---
 
 This document describes how to configure the NGINX App Protect DoS module
-> Check out the complete [NGINX Ingress Controller with App Protect DoS example for VirtualServer](https://github.com/nginxinc/kubernetes-ingress/tree/v3.0.1/examples/custom-resources/app-protect-dos) and the [NGINX Ingress Controller with App Protect DoS example for Ingress](https://github.com/nginxinc/kubernetes-ingress/tree/v3.0.1/examples/ingress-resources/app-protect-dos).
+> Check out the complete [NGINX Ingress Controller with App Protect DoS example for VirtualServer](https://github.com/nginxinc/kubernetes-ingress/tree/v3.0.2/examples/custom-resources/app-protect-dos) and the [NGINX Ingress Controller with App Protect DoS example for Ingress](https://github.com/nginxinc/kubernetes-ingress/tree/v3.0.2/examples/ingress-resources/app-protect-dos).
```
[the remark this answers is not in the document]
Comment (ADubhlaoich) [reply docs/content/app-protect-dos/configuration.md:12]: There's no noun attached to the titles, so it's unclear what you're following by clicking the link.

File: docs/content/technical-specifications.md (hunk 1 above)
Comment (vepatel) [reply docs/content/technical-specifications.md]: think we can do another once IC is out

File: docs/content/app-protect-dos/installation.md [hunk 4]
```diff
@@ -66,4 +66,4 @@ Take the steps below to set up and deploy the NGINX Ingress Controller and App P
...
 4. [Deploy the Ingress Controller](/nginx-ingress-controller/installation/installation-with-manifests/#3-deploy-the-ingress-controller).
 
-For more information, see the [Configuration guide](/nginx-ingress-controller/app-protect-dos/configuration),the [NGINX Ingress Controller with App Protect DoS example for VirtualServer](https://github.com/nginxinc/kubernetes-ingress/tree/v3.0.1/examples/custom-resources/app-protect-dos) and the [NGINX Ingress Controller with App Protect DoS example for Ingress](https://github.com/nginxinc/kubernetes-ingress/tree/v3.0.1/examples/ingress-resources/app-protect-dos).
+For more information, see the [Configuration guide](/nginx-ingress-controller/app-protect-dos/configuration),the [NGINX Ingress Controller with App Protect DoS example for VirtualServer](https://github.com/nginxinc/kubernetes-ingress/tree/v3.0.2/examples/custom-resources/app-protect-dos) and the [NGINX Ingress Controller with App Protect DoS example for Ingress](https://github.com/nginxinc/kubernetes-ingress/tree/v3.0.2/examples/ingress-resources/app-protect-dos).
```
Comment (ADubhlaoich) [inline docs/content/app-protect-dos/installation.md:69]:
```suggestion
For more information, see the [Configuration guide](/nginx-ingress-controller/app-protect-dos/configuration),the [NGINX Ingress Controller with App Protect DoS for VirtualServer](https://github.com/nginxinc/kubernetes-ingress/tree/v3.0.2/examples/custom-resources/app-protect-dos) and the [NGINX Ingress Controller with App Protect DoS for Ingress](https://github.com/nginxinc/kubernetes-ingress/tree/v3.0.2/examples/ingress-resources/app-protect-dos) examples.
```
Made an amendment to an earlier suggestion, missed something initial and created some redundancy.

File: docs/content/configuration/policy-resource.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ The Policy resource allows you to configure features like access control and rat
 
 The resource is implemented as a [Custom Resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
 
-This document is the reference documentation for the Policy resource. An example of a Policy for access control is available in our [GitHub repo](https://github.com/nginxinc/kubernetes-ingress/blob/v3.0.1/examples/custom-resources/access-control).
+This document is the reference documentation for the Policy resource. An example of a Policy for access control is available in our [GitHub repo](https://github.com/nginxinc/kubernetes-ingress/blob/v3.0.2/examples/custom-resources/access-control).
```
Comment (ADubhlaoich) [inline docs/content/configuration/policy-resource.md:16]:
```suggestion
This document is the reference documentation for the Policy resource. An example of a Policy for access control is available in our [GitHub repository](https://github.com/nginxinc/kubernetes-ingress/blob/v3.0.2/examples/custom-resources/access-control).
```

File: docs/content/configuration/policy-resource.md [hunk 6]
```diff
@@ -365,7 +365,7 @@ NGINX Plus will pass the ID of an authenticated user to the backend in the HTTP
 #### Prerequisites
 
 In order to use OIDC, you need to enable [zone synchronization](https://docs.nginx.com/nginx/admin-guide/high-availability/zone_sync/). If you don't set up zone synchronization, NGINX Plus will fail to reload.
-You also need to configure a resolver, which NGINX Plus will use to resolve the IDP authorization endpoint. You can find an example configuration [in our GitHub repo](https://github.com/nginxinc/kubernetes-ingress/blob/v3.0.1/examples/custom-resources/oidc#step-7---configure-nginx-plus-zone-synchronization-and-resolver).
+You also need to configure a resolver, which NGINX Plus will use to resolve the IDP authorization endpoint. You can find an example configuration [in our GitHub repo](https://github.com/nginxinc/kubernetes-ingress/blob/v3.0.2/examples/custom-resources/oidc#step-7---configure-nginx-plus-zone-synchronization-and-resolver).
```
Comment (ADubhlaoich) [inline docs/content/configuration/policy-resource.md:368]:
```suggestion
You also need to configure a resolver, which NGINX Plus will use to resolve the IDP authorization endpoint. You can find an example configuration [in our GitHub repository](https://github.com/nginxinc/kubernetes-ingress/blob/v3.0.2/examples/custom-resources/oidc#step-7---configure-nginx-plus-zone-synchronization-and-resolver).
```

**Review by ADubhlaoich — APPROVED** [review]
Looks good. Flagged a few small things, which are tone/style concerns.

**Comment by lucacome** [comment]
@ADubhlaoich Let's not increase the scope of this PR, we can make these changes in a dedicated PR.

File: docs/content/releases.md [hunk 7]
```diff
@@ -6,6 +6,22 @@ doctypes: ["concept"]
...
 docs: "DOCS-616"
 ---
+## NGINX Ingress Controller 3.0.2
+
+13 Feb 2023
+
+FIXES:
+* [3519](https://github.com/nginxinc/kubernetes-ingress/pull/3519) Add OnDelete to allowed strategy values
+* [3541](https://github.com/nginxinc/kubernetes-ingress/pull/3541) Ensure non-ready endpoints are not added to upstreams
+
```
Comment (lucacome) [inline docs/content/releases.md:16]:
```suggestion
* [3527](https://github.com/nginxinc/kubernetes-ingress/pull/3527) Fix controller.topologySpreadConstraints schema, thanks to [Marco Londero](https://github.com/marcuz)

```

**Review by vepatel — APPROVED** [review]

**Pull request merged**